jobs/debugging.rst

```diff
@@ -48,8 +48,8 @@ can be handy when you debug a large-scale job. Please refer to
 
 .. _TotalView user guide: https://docs.roguewave.com/en/totalview/current/html/
 
-Exiting TotalView without completing a full run of your code, may not clean up the Flux job. 
-In that case you will need to cancel the flux job manually. 
+Exiting TotalView without completing a full run of your code, may not clean
```

Comment:
I don't think a comma is needed here after `code`.

Reply:
Agree!

Reply:
Just pushed a couple fixes including dropping that comma!

Reply:
Looks like @garlick caught this in his fixes below!